.gitlab-ci.yml

```diff
@@ -258,14 +258,16 @@ variables:
   WINGET_PAT: ci.datadog-agent.winget_pat  # windows-agent
   # End aws ssm variables
   # Start vault variables
+  AGENT_API_KEY_ORG2: agent-api-key-org-2  # agent-devx-infra
+  AGENT_APP_KEY_ORG2: agent-ci-app-key-org-2  # agent-devx-infra
```

Comment:
Only added these ones and `INSTALL_SCRIPT_API_KEY_ORG2`

Reply:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓very stupid question:** Is it "ORG2" as "Our org with the id 2" (our main internal org) or another org?‏

Reply:
It is `our org with id2`. I can use another suffix if you think it's better

Reply:
I think you can leave it that way, I was just confused at first. The only problem with this name is that if one day we change the org we use, we'll need to change the name everywhere